web/gui/dashboard_info.js

```diff
@@ -3413,6 +3413,28 @@ netdataDashboard.context = {
         info: 'Size of metric samples written to disk.'
     },
 
+    // ------------------------------------------------------------------------
+    // Filesystem
+    'filesystem.deleted_objects': {
+        title : 'VFS remove',
+        info: 'This chart does not show all events that remove files from the file system, because file systems can create their own functions to remove files, it shows calls for the function <a href="https://www.kernel.org/doc/htmldocs/filesystems/API-vfs-unlink.html" target="_blank">vfs_unlink</a>. '
+    },
+
+    'filesystem.io': {
+        title : 'VFS IO',
+        info: 'Successful or failed calls to functions <a  href="https://topic.alibabacloud.com/a/kernel-state-file-operation-__-work-information-kernel_8_8_20287135.html" target="_blank">vfs_read</a> and <a href="https://topic.alibabacloud.com/a/kernel-state-file-operation-__-work-information-kernel_8_8_20287135.html" target="_blank">vfs_write</a>. This chart may not show all file system events if it uses other functions to store data on disk.'
+    },
+
+    'filesystem.io_bytes': {
+        title : 'VFS bytes written',
+        info: 'Total of bytes read or written with success using the functions <a  href="https://topic.alibabacloud.com/a/kernel-state-file-operation-__-work-information-kernel_8_8_20287135.html" target="_blank">vfs_read</a> and <a href="https://topic.alibabacloud.com/a/kernel-state-file-operation-__-work-information-kernel_8_8_20287135.html" target="_blank">vfs_write</a>.'
+    },
+
+    'filesystem.io_error': {
+        title : 'VFS IO error',
+        info: 'Failed calls to functions <a  href="https://topic.alibabacloud.com/a/kernel-state-file-operation-__-work-information-kernel_8_8_20287135.html" target="_blank">vfs_read</a> and <a href="https://topic.alibabacloud.com/a/kernel-state-file-operation-__-work-information-kernel_8_8_20287135.html" target="_blank">vfs_write</a>.'
+    },
```

Comment:
I would delete the `alibaba.cloud` links. I don't see any value for a user in the article. Do you?

Reply:
also, I suggest adding `vfs` to the chart id/context (e.g: `filesystem.io` => `filesystem.vfs_io`).

Reply:
+ additional problem of mantaining validity of such links... so we have any CI step that will check they still exist?

Reply:
<details>
<summary>We have no, there are 153 links in the dashboard_info.js</summary>

```cmd
0 gui (master)$ grep -c "href=" dashboard_info.js
153
```

</details>

Reply:
Thank you very much for your suggestions, I made all changed you required, please, take a look on them.